notebooks/2023-01-17-initial-model-ph-mm-tl-kh/model_all.ipynb

```diff
@@ -0,0 +1,1156 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #6.    print(f"Cross validation R^2 std: {cv_std}")
</pre><p> </p><p>Ahh I see why this value is different from the leave one country out validation metrics. </p><p> </p><p>This is just doing a 5-fold RepeatKFold validation on the combined 4 countries' data. </p><p> </p><p>I wonder if we should still do this, as the more conservative metric to report would be the LOCV, as it gives a better idea of how the model might generalize to a totally unseen country.</p><p> </p><p> </p><p>Think we should just remove this, as the LOCV already provides the metrics we need to evaluate this kind of approach. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/thinkingmachines/unicef-ai4d-poverty-mapping/pull/139/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='36'/>